app/templates/live.hbs

```diff
@@ -1,6 +1,20 @@
-<LiveHeader
-  @tabs={{this.TABS}}
-  @activeTab={{this.activeTab}}
-  @tabHandler={{this.tabHandler}}
-/>
-<p>{{this.activeTab}} Component</p>
\ No newline at end of file
+{{page-title 'Live'}}
+
+<div id='live' data-test-live>
+  <LiveHeader
+    @tabs={{this.TABS}}
+    @activeTab={{this.activeTab}}
+    @tabHandler={{this.tabHandler}}
+  />
+  {{#if (eq this.activeTab 'Screenshare')}}
+    <div class='live__video-container' data-test-video-container>
+      {{#if this.isLoading}}
+        <Reusables::Skeleton />
+      {{else}}
+        <VideoScreen @src='/assets/birds_footage.mp4' />
```

Comment:
You can add the TODO text here. as the source is dummy as of now

Reply:
Added